scripts/builtin/lmDS.dml

```diff
@@ -0,0 +1,139 @@
+#-------------------------------------------------------------
+#
+# Copyright 2019 Graz University of Technology
```

Comment:
Use the modifications-license as the original linregDS was copied from Apache SystemML.